ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityTextAnswer.component.ts

```diff
@@ -83,12 +83,12 @@ export class ActivityTextAnswer implements OnChanges, OnInit {
     }
 
     public isTextQuestion(block: ActivityTextQuestionBlock): boolean {
-        return !block.inputType || block.inputType.toLowerCase() === InputType.Text
-            || block.inputType.toLowerCase() === InputType.Signature;
+        return !block.inputType || block.inputType === InputType.Text
+            || block.inputType === InputType.Signature;
```

Comment:
Since I needed to add the `Email` InputType, I took the opportunity to just use it as straight up `enum` (i.e., no need to compare strings)